src/components/icon/Icon.tsx

```diff
@@ -0,0 +1,10 @@
+import { allIcons, IconName, IconProps } from "@components/svg/Icons";
+
+interface IconComponentProps extends IconProps {
+  name: IconName;
+}
+
+/**
+ * Renders an SVG icon from our internal icons library. All icons are SVG markup as JSX, allowing us to set currentColor to colour them.
+ */
+export const Icon = ({ name, ...iconProps }: IconComponentProps) => allIcons[name](iconProps);
```

Comment:
thought: Would we want to deprecate the use of `svg/Icon` - as a thought experiment, as a new developer to the codebase, I am not sure I would know which to use?

Reply:
Good point. I liked the idea of keeping this file because it separates content from execution, but it might be more wholesale to keep everything together in one place, particularly because the `Icon` implementation is so [elegantly] bare.